configs/tests/test_cpu_relax.yml

```diff
@@ -0,0 +1,30 @@
+dask:
+  partitions: -1 # if -1, 1 row per surface otherwise number of partitions
+
+memory_cache_location: '/home/jovyan/shared-scratch/catlas/test_cache_cpu_relax'
```

Comment:
do these really need their own caches?

Reply:
Yeah, I want to wipe the cache and rerun things as needed, and I never want to keep any of these caches